checks/check_extra726

```diff
@@ -47,7 +47,7 @@ extra726(){
         textInfo "$REGION: Trusted Advisor check $TA_CHECKS_NAME is in warning state $QUERY_TA_CHECK_RESULT" "$REGION" "$TA_CHECKS_NAME"
         ;;
       "not_available")
-        textInfo "$REGION: Trusted Advisor check $TA_CHECKS_NAME is in not_available state $QUERY_TA_CHECK_RESULT" "u$REGION" "$TA_CHECKS_NAME"
+        textInfo "$REGION: Trusted Advisor check $TA_CHECKS_NAME is in not_available state $QUERY_TA_CHECK_RESULT" "$REGION" "$TA_CHECKS_NAME"
```

Comment:
There was a typo in the region name which was prefixed by u

Reply:
thanks